lib-extra/build.gradle

```diff
@@ -22,6 +22,5 @@ dependencies {
 }
 
 // we'll hold the core lib to a high standard
-findbugs {
-	reportLevel = 'low' // low|medium|high (low = sensitive to even minor mistakes)
+findbugs { reportLevel = 'low' // low|medium|high (low = sensitive to even minor mistakes)
 }
```

Comment:
It's not clear to me why Greclipse formatted this section like this.

Can we prevent it with a setting?